lib/money/rates_store/eu_central_bank_historical_data_support.rb

```diff
@@ -20,7 +20,7 @@ def transaction(force_sync = false, &block)
         # locking within the same thread.
         force_sync = false
       end
-      if !force_sync && (@in_transaction || options[:without_mutex])
+      if !@mutex.respond_to?(:synchronize) || (!force_sync && (@in_transaction))
```

Comment:
Can we just check for `@mutex` not to be `nil` instead? Also the last set of parens is unnecessary